docs/source/en/training/distributed_inference.md

```diff
@@ -52,6 +52,62 @@ To learn more, take a look at the [Distributed Inference with 🤗 Accelerate](h
 
 </Tip>
 
+### Using `device_map`
+
+<Tip warning={true}>
+
+🧪 This feature is experimental and its APIs might change in future. 
+
+</Tip>
+
+With 🤗 Accelerate, you can make use of a supported `device_map` strategy to distribute the model-level components of a pipeline across multiple devices. The section below explains a situation where performing distributed inference via `device_map` could be useful.
```

Comment:
```suggestion
With Accelerate, you can use the `device_map` to determine how to distribute the model-level components of a pipeline across multiple devices. This is useful in situations where you have more than one GPU.
```

Reply:
`model-level components` is a preference as I think it's clearer.